utils/device_config.py

```diff
@@ -0,0 +1,66 @@
+import yaml
+import struct
+
+class DeviceConfig:
+  def __init__(self, path):
+    with open(path) as config_file:
+      self.yaml_config = yaml.load(config_file, Loader=yaml.FullLoader)
+  
+  def get_attribute(self, attribute):
+    return self.yaml_config[attribute]
+
+  def get_device_id(self):
+    return self.get_attribute('device_id')
+  
+  def get_sensors(self):
+    return self.get_attribute('sensors')
+
+  def get_values(self, sensor_name):
+    sensors = self.get_sensors()
+    for sensor in sensors:
+      if sensor['name'] is sensor_name:
+        return sensor['values']
+    raise Exception("bad sensor name")
+
+def number_to_bytes(number, data_type) -> bytearray:
```

Comment:
I think it would be more robust to extract the numbers to words (e.g. SHORT_BYTE_SIZE) to allow for an unanticipated change down the line.

Low priority, though